test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/helper/TypeCheck.java

```diff
@@ -1,18 +1,20 @@
 package uk.gov.justice.services.test.utils.core.helper;
 
-import uk.gov.justice.services.test.utils.core.random.Generator;
-
-import java.util.function.Function;
-
 import static java.util.Objects.requireNonNull;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.Is.is;
 
+import uk.gov.justice.services.test.utils.core.random.Generator;
```

Comment:
Do we still have issues with imports being reordered, is everyone using the same style guide?